From 6a44fbaf16c51a5b3803ec9948a131a88b6c350d Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Thu, 18 Dec 2025 11:04:25 +0100
Subject: [PATCH] Make mir-renderer-gl-dev dependency optional

Mir 2.25 has "roll[ed] mir-renderer-gl (et alia) into mirplatform", as
per the changelog.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d4ebadc9..b934de2a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,7 +86,7 @@ pkg_check_modules(MIROIL miroil REQUIRED)
 else()
 pkg_check_modules(MIRCLIENT mirclient>=0.26 REQUIRED)
 endif()
-pkg_check_modules(MIRRENDERERGLDEV mir-renderer-gl-dev>=0.26 REQUIRED)
+pkg_check_modules(MIRRENDERERGLDEV mir-renderer-gl-dev>=0.26)
 pkg_check_modules(MIRAL miral>=1.1.0 REQUIRED)
 pkg_check_modules(MIRWAYLAND mirwayland REQUIRED)
 pkg_check_modules(WAYLAND_CLIENT REQUIRED wayland-client)
-- 
2.52.0

